src/runtime/utils.ts

```diff
@@ -321,13 +321,15 @@ function isRootRedirectOptions(rootRedirect: unknown): rootRedirect is RootRedir
 const useRedirectState = () => useState<string>(NUXT_I18N_MODULE_ID + ':redirect', () => '')
 
 type NavigateArgs = {
+  nuxtApp: NuxtApp
   i18n: I18n
   redirectPath: string
   locale: string
   route: RouteLocationNormalized | RouteLocationNormalizedLoaded
 }
 
 function _navigate(redirectPath: string, status: number) {
+  console.log(redirectPath, status)
```

Comment:
```suggestion
```

Reply:
Maybe you forgot to delete the debug log, we should remove the console log. :)
If we want to output debug logs, we should use them with the `__DEBUG__` flag.

Reply:
Ah! You're right I forgot to remove it 😅